src/main/java/com/google/step/finscholar/firebase/FirebaseStorageManager.java

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.finscholar.firebase;
+
+import javax.servlet.Servlet;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.CollectionReference;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.DocumentSnapshot;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.FirebaseException;
+import com.google.gson.Gson;
+import com.google.step.finscholar.data.ServletConstantValues;
+
+
+/** This class handles writing to and reading from a Cloud Firestore database. */
+public class FirebaseStorageManager {
+  /** This is used to convert Objects into JSON strings to send to the frontend. */
+  private static Gson gson = new Gson();
+  
+  /** 
+   * This method stores an object as a DocumentReference (a data point) 
+   *   inside a Collection (of DocumentReferences) in Firestore. 
+   * @param database - This is the database I want to write my data point to.
+   * @param collectionToWriteTo - This is the collection of Documents I want to write to.
+   * @param object - The object that will be stored as a new Document in the collectionToWriteTo.
+   *   It doesn't matter what type the object is as DocumentReference.set() is type-agnostic and will
+   *   store any serializable object. This object is required to have a public, no-parameter constructor.
+   *   Also, the instance variables for that object should not be declared as final. 
+   *   There are no other requirements for the object.
+   */
+  public static void storeDocument(Firestore database, String collectionToWriteTo, Object object) throws FirebaseException {
+    // Access the correct collection.
+    CollectionReference collectionRef = database.collection(collectionToWriteTo);
+    
+    // Create a new document in the collection.
+    DocumentReference documentRef = collectionRef.document();
+
+    // Update the document with a new object.
+    ApiFuture<WriteResult> future = documentRef.set(object);
+    try {
+      // Print to console which Collection I added to and when.
+      System.out.println(ServletConstantValues.NEW_DOCUMENT_ADDED + 
```

Comment:
Does App Engine log this? We may need to use a different logging API instead of this.